components/shared/profile/Schedule.tsx

```diff
@@ -1,12 +1,35 @@
 import { IUser } from "@/database/models/user.model";
 import React from "react";
-
+import userCheck from "@/public/icons/userCheck.svg";
```

Comment:
![image](https://github.com/agordon123/git_note/assets/4108484/5da2dcd3-ae1f-4de4-82df-afb6ded8890c)


We have to make sure the casing is correct. This will not run on a linux server (like vercel or 99% of places where you'll deploy a web app).

The file is named "usercheck" not "userCheck"
```suggestion
import userCheck from "@/public/icons/usercheck.svg";
```